GitHubProject/app/src/main/java/com/example/githubproject/home/HomeFragment.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.githubproject.home
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.example.githubproject.R
+
+class HomeFragment : Fragment(){
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val view: View = inflater.inflate(R.layout.fragment_home, container, false)
+
+
+
+        return view
+    }
```

Comment:
```suggestion
    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        return inflater.inflate(R.layout.fragment_home, container, false)
    }
```
이렇게 인라인으로 줄일수 있어.
하지만 우린 데이터바인딩을 배웠으니까 데이터바인딩 코드로 view를 반환해보자.

Reply:
다른 곳도 마찬가지로!